includes/free-trial/class-wc-calypso-bridge-free-trial-orders-notice.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Class WC_Calypso_Bridge_Free_Trial_Orders_Notice.
+ *
+ * @since   1.9.16
+ * @version 1.9.16
```

Comment:
What about the versions? We're already in 2.0.3

Reply:
Updated in https://github.com/Automattic/wc-calypso-bridge/pull/936/commits/b3e521c1c70a417679a0493224f9c2cf703f3f92

Regarding version numbers, I think we should have a git hook to check class versions 🤔

Reply:
Hi @PanosSynetos ✋ 

Could you take a look at it one more time please? I've updated the version.

Reply:
Hi @PanosSynetos ✋ 

Could you please take a look at the change request? Versions have been updated.

Reply:
Good :) Thanks and sorry I missed your ping yesterday

Reply:
Thank you 😄